electron/src/main.ts

```diff
@@ -171,6 +171,7 @@ const bindIpcEvents = (): void => {
   });
 
   ipcMain.on(EVENT_TYPE.ACTION.NOTIFICATION_CLICK, () => WindowManager.showPrimaryWindow());
+  ipcMain.on(EVENT_TYPE.WEBAPP.APP_LOADED, WindowManager.flushActionsQueue);
```

Comment:
Point-free can be risky if, in the future, the `flushActionQueue` starts to take an argument. 

These are quite hard bugs to fix

```suggestion
  ipcMain.on(EVENT_TYPE.WEBAPP.APP_LOADED, () => WindowManager.flushActionsQueue());
```